pkgs/desktops/pantheon/apps/switchboard-plugs/mouse-touchpad/default.nix

```diff
@@ -60,12 +60,6 @@ stdenv.mkDerivation rec {
       src = ./fix-paths.patch;
       touchegg = touchegg;
     })
-    # Upstream code not respecting our localedir
-    # https://github.com/elementary/switchboard-plug-mouse-touchpad/pull/185
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
https://github.com/NixOS/nixpkgs/pull/139163